indico/modules/auth/templates/users/accounts.html

```diff
@@ -0,0 +1,150 @@
+            <div class="i-box-header">
+                <div class="i-box-title">
+                    {%- trans %}External Accounts{% endtrans -%}
+                </div>
+            </div>
+            <div class="i-box-content">
+                <p>
+                    {% trans %}External accounts linked with your Indico user.{% endtrans %}
+                </p>
+                <ul class="group-list with-buttons">
+                    {% if user.external_identities %}
+                        {% for identity in user.external_identities | sort(attribute='last_login_dt', reverse=True) %}
+                            <li>
+                                <span class="account-identity">
+                                    {{ provider_titles[identity.provider] }}
+                                </span>
+                                <small>
+                                    ({{ identity.identifier }})
+                                </small>
+                                <span class="right">
+                                    <span class="account-identity-info">
+                                        <span>
+                                            <span class="label">
+                                                {% trans %}Last login:{% endtrans %}
+                                            </span>
+                                            <span class="content">
+                                                {{ identity.last_login_dt | format_datetime('short') }}
+                                            </span>
+                                        </span>
+                                    </span>
+                                    {% if identity.id == session.login_identity %}
+                                        <a class="i-button disabled"
+                                            title="{% trans %}The account used to log in cannot be unlinked{% endtrans %}">
+                                    {% else %}
+                                        <a href="{{ url_for('auth.remove-account', identity) }}" class="i-button right">
+                                    {% endif %}
+                                        {% trans %}Unlink{% endtrans %}
+                                    </a>
+                                </span>
+                            </li>
+                        {% endfor %}
+                    {% else %}
+                    <li>
+                        <span class="account-no-identities">
+                        {% trans %}You have no external accounts connected{% endtrans %}
+                        </span>
+                    </li>
+                </ul>
+                {% endif %}
+            </div>
+        </div>
+    </div>
+
+    <script>
+    $(function() {
+        $('.js-show-form').on('click', function(e) {
+            e.preventDefault();
+            $('#no-local-acccount').hide();
+            $('#credentials-form').removeClass('hidden');
```

Comment:
`.show()` doesn't unhide our `.hidden` class. Do you know a better way of having things hidden on page load that plays better with jQuery?